terraform/central-account/security_group.tf

```diff
@@ -14,26 +14,8 @@ resource "aws_security_group_rule" "internal_ingress_all_self" {
   cidr_blocks       = [var.vpc_cidr]
 }
 
-resource "aws_security_group_rule" "external_ingress_ssh" {
```

Comment:
This did not spark joy. Thank you for removing it.

Reply:
SSM FTW :)

Reply:
Nice